src/main/java/core/basesyntax/service/FileWriteImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service;
+
+import core.basesyntax.service.impl.FileWrite;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileWriteImpl implements FileWrite {
+    @Override
+    public void write(String result, String nameFileWrite) {
+        try {
+            Files.write(Paths.get(nameFileWrite), result.getBytes());
+        } catch (IOException e) {
+            throw new RuntimeException("Cant write file " + nameFileWrite, e);
```

Comment:
The exception message should be more informative and consistent with the rest of the project. Consider changing it to include the action that was being performed, like "Can't write to file: " followed by the file path.